src/bindings/xfconf/README.md

```diff
@@ -0,0 +1,111 @@
+- infos =
+- infos/author = Richard Stöckl <e11908080@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/status = experimental maintained
+- infos/provides =
+- infos/description = Elektra Bindings for xfconf
+
+## Introduction
+
+These bindings provide an implementation of the xfconf header files such that libelektra can be used as a drop-in replacement for xfconf.
+In other words, it makes applications which rely on xfconf for their configuration use libelektra without the necessity for modifying their source code or even recompiling them.
+
+For further understanding how xfconf is structured, please refer to the [xfconf plugin](../../../src/plugins/xfconf/README.md#xfconf-terminology).
+
+todo.
+
+## Files
+
+- The header files of this implementation are provided by xfconf and are not included here.
+- The resulting library is called `libxfconfbinding.so`
+
+## Classes
+
+Explain which [classes](/doc/help/elektra-glossary.md) the library contains.
+Add a link to the API documentation.
+todo.
+
+## Dependencies
+
+### Compile-Time
+
+The xfconf library from the XFCE project is the main dependency of this plugin.
+Usually, this library is called something such as `xfconf` (Arch, Fedora, `xfconf-devel` for compiling), `libxfconf-0` (Debian, `libxfconf-0-dev` for compmiling) or `xfce4-conf` (FreeBSD) in the package manager.
```

Comment:
Had to upgrade to Ubuntu 22.04 because of the xfconf 4.16 requirement.
```
sudo apt-get install xfconf glib-2.0 libxfconf-0-dev
```
Without `libxfconf-0-dev` i would still get:
```
-- Could NOT find Xfconf (missing: XFCONF_LIBRARY XFCONF_INCLUDE_DIR)
-- Exclude binding xfconf because xfconf >= 4.16 required for xfconf backend
```

@Eiskasten maybe include this for Ubuntu users?

Reply:
And of course had to enable `xfconf` in bindings and plugins

Reply:
Ok, so the dev package is the same as in Debian. I will include that.